### PR TITLE
example/basic: make basic example work

### PR DESCRIPTION
**Pull request by clsung**
- use stdout as exportor (otherwise no output when you run it)

### CODE REVIEW

File: example/basic/main.go [hunk 1]
```diff
@@ -16,25 +16,45 @@ package main
...
+	if err != nil {
+		log.Panicf("failed to initialize trace provider %v\n", err)
+	}
+	global.SetTraceProvider(tp)
+}
+
 func main() {
+	initTracer()
+	tracer := global.TraceProvider().GetTracer("ex.com/basic")
+	meter := global.MeterProvider().GetMeter("ex.com/basic") // TODO: should share resources ^^^?
```
Comment (rghetia) [inline example/basic/main.go]: please add a note that Meter doesn't work yet.

Comment (clsung) [reply example/basic/main.go]: Just rephrase TODO, thanks.

Comment (jmacd) [reply example/basic/main.go]: FYI there is a Meter stdout exporter in #265, which also adds to this example.
It's fine to remove the note about flush.

**Comment by jmacd** [comment]
For me, the TODO about resources is a spec-level "nice-to-have". It should be relatively easy to add resources to the meter and tracer in a coordinated way, but we haven't specified a way to do that. I believe the right approach would be to combine the two interfaces and support a common method for adding resources to a Tracer/Meter combo.

**Pull request merged**